lib/commands/dev/prepackage.ts

```diff
@@ -28,11 +29,12 @@ export class PrePackageCommand implements ICommand {
 			this.$logger.info("Unpacking app resources.");
 			this.$templatesService.unpackAppResources().wait();
 			this.$logger.info("Downloading cordova migration data.");
-			this.$cordovaMigrationService.downloadCordovaMigrationData().wait();
-			//Cordova files have to be downloaded after cordova migration data so we know which cordova versions we support
+			this.$cordovaMigrationService.downloadMigrationData().wait();
+			// Cordova files have to be downloaded after cordova migration data so we know which cordova versions we support
 			this.$logger.info("Downloading cordova.js files.");
 			this.$resourceDownloader.downloadCordovaJsFiles().wait();
-
+			this.$logger.info("Downloading nativescript migration data.")
+			this.$nativeScriptMigrationService.downloadMigrationData().wait();
```

Comment:
Well it's just 5 mb and with NativeScript 1.1 we will not have to do it, so I've decided to keep current processes and have the migration data as part of our package.